pwa/src/components/common/arrayInput.tsx

```diff
@@ -24,22 +24,22 @@ export function ArrayInputComponent(props: ArrayInputProps) {
       <div id={`new${_.upperFirst(props.id)}`}>
         {props.data !== undefined &&
           props.data !== null &&
-          props.data.map((item) => (
-            <div key={item} className={`row ${item}`}>
+          props.data.map((item, idx) => (
+            <div key={idx} className={`row ${item}`}>
               <div className="col-5">
                 <div className="form-group">
                   <input
                     type="text"
                     id="value"
                     name={`${props.id}[${item}]`}
-                    defaultValue={item}
+                    defaultValue={item.toString()}
```

Comment:
Je zou hier bijvoorbeeld dan ook geen `.toString()` nodig hebben wanneer je niet `:any` had gebruikt maar vooraf had gedefinieerd om wat voor datatype het ging.

Nu zou het kunnen zijn dat `item` een data type is waar je niet `toString` op kan zeggen waardoor je een error krijgt.